### PR TITLE
Add google analytics

### LINKED ISSUE #74: Add google analytics

### PR DESCRIPTION
**Pull request by jackreimers**
Closes #74 

Added google analytics using https://github.com/isc30/blazor-analytics 
Google analytics property set up in the SSW Google account

### CODE REVIEW

**Comment by Harry-Ross** [comment]
Which SSW Google account is it set up with?

**Comment by jackreimers** [comment]
google@ssw.com.au

File: src/WebUI/Program.cs [hunk 1]
```diff
@@ -22,6 +23,7 @@
 builder.Services.AddSingleton<MarkdigPipelineService>();
 builder.Services.AddScoped<SswRulesGptDialogService>();
 builder.Services.AddScoped<ApiKeyValidationService>();
+builder.Services.AddGoogleAnalytics("G-239HQG61FX");
```
Comment (Harry-Ross) [inline src/WebUI/Program.cs:26]: Can we move this to a config variable?

Comment (jackreimers) [reply src/WebUI/Program.cs]: @Harry-Ross I've moved the ID into appsettings

**Review by Harry-Ross — APPROVED** [review]

**Pull request merged**